source/content/modules-known-issues.md

```diff
@@ -311,6 +311,23 @@ ___
 **Solution**: You can try to patch the [permission check in the module](https://github.com/thephpleague/oauth2-server/blob/e184691ded987c00966e341ac09c46ceeae0b27f/src/CryptKey.php#L51). The alternative is to use off-site key management tools like [Lockr](https://www.drupal.org/project/lockr)
 ___
 
+## [Update Manager](https://www.drupal.org/docs/8/core/modules/update-manager) - Drupal 8 (core)
+
+<ReviewDate date="2020-03-26" />
+
+**Issue**: Sometimes, after a fresh system install, the **Manage** > **Extend** > **+ Install new module** and **Manage** > **Appearance** > **+ Install new theme** buttons are missing.
+
+This is a known bug in Drupal 8. See [#3033480](https://www.drupal.org/project/drupal/issues/3033480) for more information.
+
+**Solution**: 
+
+1. Navigate to the **Manage** > **Extend** page at `/admin/modules`.  
+1. Click the **Uninstall** tab, and uninstall the Update Manager module. 
+1. Click the **List** tab and re-install the Update Manager. 
+1. [Rebuild the cache](/clear-caches). 
```

Comment:
Not sure if I'm just being pedantic or not, but we talk about clearing the cache on that page and in general. _Technically_ it's not rebuilt until more traffic happens.

Reply:
I believe that in Drupal 7, a cache clear erases cached objects, which are later rebuilt on page loads, whereas in Drupal 8, a cache rebuild by-and-by-large rebuilds the objects at the time of the rebuild command. This is not strictly true for all objects, e.g. there are some twig cache files etc. that still have the erase now / rebuild later behavior.

Reply:
Now there is this page on drupal.org that's all about "cache clear":

https://www.drupal.org/docs/user_guide/en/prevent-cache-clear.html

However, I don't think we can take it as entirely authoritative, as it uses the Drupal 7 version of the Drush command (cache:clear instead of cache:rebuild). However, I think that the "clear" nomenclature is still common enough in Drupal documentation that I have no objection to saying "clear" instead of "rebuild" for consistency

Reply:
Changed.